templates/CRM/Birthdays/Form/Settings.tpl

```diff
@@ -0,0 +1,196 @@
+    {ts}will <u>send an actual email</u>skipping db redirection!{/ts}
+</div>
+<br>
+<h2>
+    {ts}Step 8: Testing{/ts}
+</h2>
+<div id="help">
+    {ts}Check if you want to use the special debug mode described in the next chapter to avoid traces in your system{/ts}<br>
+    {ts}Again use the topbar to navigate to: "Administer" --> "Administration Console" -->{/ts} "
+    <a href="{crmURL p='/civicrm/admin/job' q='reset=1'}">{ts}Scheduled Jobs{/ts}</a>".<br>
+    {ts}Find the previous created birthday job in this list{/ts}<br>
+    {ts}Click: "more"{/ts}<br>
+    {ts}Click: "Execute Now" <b>(Warning: <u>Actual emails</u> will be sent to the contacts email addresses if everything works as expected!)</b>{/ts}<br>
+    {ts}Your previoulsy selected contacts should now receive an email with birthday greetings{/ts}
+    {ts}Make sure to check what type of activity description had been added to your selected contacts.{/ts}
+    <br>
+    {ts}If you are familiar with executing API commands you also can use{/ts}
+    <a href="{crmURL p='/civicrm/api4#/explorer/Birthdays/sendGreetings' q=''}">APIv4</a>
+    {ts}to trigger sending of mails.{/ts}
+</div>
+<br>
+
+
+<br>
+<h2>
+    {ts}Test / debug mode (!! Sysadmins only !!){/ts}
+</h2>
+<div id="help"><b>{ts}A debug mode can be set programatically:{/ts}</b><br>
+    {ts}Navigate to:{/ts} <code>/de.systopia.birthdays/CRM/Birthdays/BirthdayContacts.php</code><br>
+    {ts}Set:<code>$is_debug_email = 'all-birthday-mails-go-to@this-domain.com';</code>using an code editor.{/ts}
+    <p>{ts}This changes the extensions behavior to:{/ts}</p>
+    <ul> <!-- Please sync this documentation with mentioned php class path-->
+        <li>{ts}Adding "successful" or "failed" activities will be suppressed{/ts}</li>
+        <li>{ts}All emails will be redirected to this email set above{/ts}</li>
+        <li>{ts}A filter is de-activated which selects the first 10 contacts/mails where a birth date is set{/ts}</li>
+    </ul>
+    <br>
+    {ts}Set an empty string <code>$is_debug_email = '';</code> to de-activate debug mode.{/ts}
+</div>
+<br>
+<h2>
+    {ts}Final activation{/ts}
+</h2>
+<div id="help">
+    {ts}Again use the topbar to navigate to: "Administer" --> "Administration Console" -->{/ts} "
+    <a href="{crmURL p='/civicrm/admin/job' q='reset=1'}">{ts}Scheduled Jobs{/ts}</a>".<br>
+    {ts}Again find the previous created birthday job in this list{/ts}<br>
+    {ts}Again click: "more"{/ts}<br>
+    {ts}Is this Scheduled Job active?: Set a first execution date and time.{/ts}<br>
+    {ts}Click: "Save"{/ts}<br>
+    {ts}Again find the previous created birthday job in this list{/ts}<br>
+    {ts}Again click: "more"{/ts}<br>
+    {ts}Click: "Enable". This enables the daily sending of birthday mails for your instance.{/ts}<br>
+</div>
+<br>
+<hr>
+<br>
+<h2>
+    {ts}Extension development{/ts}
```

Comment:
In general, there's way too much text on this page that would much better fit in the docs. Let's not confuse users with long explanations here, especially not with developer docs.

Reply:
I kinda like the blend-in-docs style as it allows linking to specific paths instead of describing everything in a more complicated way in external docs. 
Any specific suggestion of sections to move to external docs?